lib/logux/logging.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+module Logux
+  module Logging
+    def self.info(message, data = '')
+      msg = message.light_yellow
+      if data.present? && data.is_a?(Hash)
+        msg += ': '.light_yellow +
+               JSON.pretty_generate(data).light_white.on_light_blue
+      end
+
+      logger.info(msg)
+    end
+
+    def self.error(message)
```

Comment:
If we aren't changing the default behavior of this methods, will be better to use `delegate` from `ActionSupport`.